File: features/ads/src/main/java/com/mmt/ads/models/AdsType.java
package com.mmt.ads.models;

public enum AdsType {
    BANNER_BOTTOM("banner_bottom"),
    BANNER_EMPTY_SCREEN("banner_empty_screen"),
    BANNER_EXIT_DIALOG("banner_exit_dialog"),
    INTERSTITIAL_OPA("interstitial_opa"),
    REWARDED_AD("rewarded_ad"),
    INTERSTITIAL("interstitial");

    final String value;

    public String getValue() {
        return value;
    }

    AdsType(String value) {
        this.value = value;
    }
}

File: features/ads/src/main/java/com/mmt/ads/models/AdsType.kt
package com.mmt.ads.models

enum class AdsType(var value: String) {
    BANNER_ALL("banner_all"),
    NATIVE_ALL("native_all"),
    NATIVE_LIST("native_list"),
    NATIVE_TAB_HOME("native_tab_home"),
    NATIVE_EXIT_DIALOG("native_exit_dialog"),
    NATIVE_EMPTY_SCREEN("native_empty_screen"),
    NATIVE_LYRICS_EMPTY("native_lyrics_empty"),
    NATIVE_LYRICS_DIALOG("native_lyrics_dialog"),
    INTERSTITIAL_OPA("interstitial_opa"),
    APP_OPEN_ADS("app_open_ads"),

    INTERSTITIAL("interstitial")
}
